cmd/agent/app/integrations.go

```diff
@@ -268,12 +318,25 @@ func installTuf(cmd *cobra.Command, args []string) error {
 	}
 
 	intVer := strings.Split(args[0], "==")
-	integration := strings.TrimSpace(intVer[0])
+	integration := strings.Replace(strings.TrimSpace(intVer[0]), "_", "-", -1)
 	if integration == "datadog-checks-base" {
-		return fmt.Errorf("cannot upgrade datadog-checks-base")
+		return fmt.Errorf("this command does not allow installing datadog-checks-base")
```

Comment:
```suggestion
		return fmt.Errorf("this command does not allow updating datadog-checks-base")
```

Reply:
But datadog-checks-base is already installed on disk, hence it's not accurate. But do what you like!